helper/include/odp/helper/gtpu.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2020, Linaro Limited
```

Comment:
Copyright attribution seems to be wrong here and also in the new igmp header file.

Reply:
Ack